### PR TITLE
Fix several translation issues in properties file

### PR DESCRIPTION
**Pull request by AlexGacon**
Fix several encoding issues in translation of properties file 
 

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md).
- [ ] I have sent a [Contribution Licence Agreement](https://docs.geoserver.org/latest/en/developer/policies/committing.html) (not required for small changes, e.g., fixing typos in documentation).
- [x] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
- [x] All the build checks are green ([see automated QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html)).

For core and extension modules:

- [ ] New unit tests have been added covering the changes.
- [ ] [Documentation](https://github.com/geoserver/geoserver/tree/main/doc/en/user/source) has been updated (if change is visible to end users).
- [ ] The [REST API docs](https://github.com/geoserver/geoserver/tree/main/doc/en/api/1.0.0) have been updated (when changing configuration objects or the REST controllers).
- [ ] There is an issue in the [GeoServer Jira](https://osgeo-org.atlassian.net/browse/GEOS/summary) (except for changes that do not affect administrators or end users in any way).
- [ ] Commit message(s) must be in the form ``[GEOS-XYZWV] Title of the Jira ticket``.
- [ ] Bug fixes and small new features are presented as a single commit.
- [ ] Each commit has a single objective (if there are multiple commits, each has a separate JIRA ticket describing its goal).

### CODE REVIEW

**Comment by AlexGacon** [comment]
It is ready to merge.

Le mer. 9 févr. 2022 à 15:19, Andrea Aime ***@***.***> a
écrit :

File: build/transifex/pom.xml [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+            </configuration>
+          </execution>
+          <execution>
+            <id>wmtsstores</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+            <configuration>
+              <inputSpec>${project.basedir}/api/1.0.0/wmtsstores.yaml</inputSpec>
```
Comment (aaime) [inline build/transifex/pom.xml:455]: What's happening in this POM? 
Are you trying to provide translations for the REST API definitions as well?

Comment (AlexGacon) [reply build/transifex/pom.xml:455]: No I simply copy/paste the POM from the documentation projet and didn't replace it. I will remove it from now.

File: build/transifex/.tx/config [hunk 2]
```diff
@@ -107,4 +107,82 @@ minimum_perc = 10
 file_filter  = ../../src/extension/netcdf-out/src/main/resources/GeoServerApplication_<lang>.properties
 source_file  = ../../src/extension/netcdf-out/src/main/resources/GeoServerApplication.properties
 type         = PROPERTIES
-minimum_perc = 10
\ No newline at end of file
+minimum_perc = 10
```
[the remark this answers is not in the document]
Comment (AlexGacon) [reply build/transifex/.tx/config:110]: I hesitated but some of the files inside the repo were below 10 percent so I prefered to have them updated.

File: build/transifex/pom.xml (hunk 1 above)
Comment (aaime) [reply build/transifex/pom.xml:455]: I was checking if I could merge, but the pom.xml seems to be still there.

File: build/transifex/.tx/config (hunk 2 above)
[the remark this answers is not in the document]
Comment (aaime) [reply build/transifex/.tx/config:110]: Makes sense, thank you.

File: build/transifex/pom.xml (hunk 1 above)
Comment (AlexGacon) [reply build/transifex/pom.xml]: Sorry Andrea. I just did it.

**Comment by aaime** [comment]
Squash-merged into main. Thanks!

**Pull request merged**